<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgContactAuthors</class>
 <widget class="QDialog" name="DlgContactAuthors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact QuickFit authors</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../lib/lib.qrc">
    <normaloff>:/lib/mail.png</normaloff>:/lib/mail.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Your Name:</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Your e-mail adress:</string>
     </property>
     <property name="buddy">
      <cstring>edtEMail</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtEMail"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>send to:</string>
     </property>
     <property name="buddy">
      <cstring>cmbSendTo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cmbSendTo">
       <item>
        <property name="text">
         <string>QuickFit 3 authors</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QuickFit 3 mailinglist</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labMailingList">
       <property name="text">
        <string>&lt;small&gt;&lt;b&gt;Note:&lt;/b&gt; You have to first &lt;a href=&quot;mailto:quickfit-request@dkfz.de?subject=subscribe&amp;amp;body=subscribe&quot;&gt;subscribe to the mailinglist&lt;/a&gt; to be allowed to send messages to it!&lt;/small&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>subject:</string>
     </property>
     <property name="buddy">
      <cstring>cmbSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbSubject">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>comment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>error report</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>question</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>feature request</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>message:</string>
     </property>
     <property name="buddy">
      <cstring>edtMessage</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPlainTextEdit" name="edtMessage"/>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="labWebpage">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>QuickFit version:</string>
     </property>
     <property name="buddy">
      <cstring>edtVersion</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="edtVersion"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgContactAuthors</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgContactAuthors</receiver>
   <slot>acceptDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>DlgContactAuthors</receiver>
   <slot>openHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptDialog()</slot>
  <slot>openHelp()</slot>
 </slots>
</ui>
